notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data.to_csv("../data/featsel/Citywide_Survey_featsel.csv")
</pre><p>the feature-selected data from the&nbsp;<em>update shape&nbsp;</em>process is located in this directory path. Please modify to the&nbsp;<em>processed</em>&nbsp;folder's directory.&nbsp;<strong>not to mention returning the previous file from the "featsel" folder.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Updated to _processed_ folder path.